### ISSUE TITLE
MD028/no-blanks-blockquote - invalid linting warning 

### ISSUE THREAD

**Issue by cjwijtmans** [DavidAnson/vscode-markdownlint#124]
example:

```markdown
> `/var/db/repos/localrepo/profiles/repo_name`
>
> ```bash
> localrepo
> ```
<!-- MD028/no-blanks-blockquote -->
> `/var/db/repos/localrepo/metadata/layout.conf`
>
> ```ini
> masters = gentoo
> auto-sync = false
> ```
<!-- MD028/no-blanks-blockquote -->
> `/etc/portage/repos.conf/localrepo.conf`
>
> ```ini
> [localrepo]
> location = /var/db/repos/localrepo
> ```
```

(replace `<!-- MD028/no-blanks-blockquote -->` with blank line)

**Comment by DavidAnson** [comment]
Could you please say what you think is invalid or not working correctly?

Here is the documentation for that rule: ￼https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md028---blank-line-inside-blockquote

Running it against your example seems to report what it says it reports:￼ https://dlaa.me/markdownlint/#%25m%23%20Issue%20124%0A%0A%3E%20%60%2Fvar%2Fdb%2Frepos%2Flocalrepo%2Fprofiles%2Frepo_name%60%0A%3E%0A%3E%20%60%60%60bash%0A%3E%20localrepo%0A%3E%20%60%60%60%0A%0A%3E%20%60%2Fvar%2Fdb%2Frepos%2Flocalrepo%2Fmetadata%2Flayout.conf%60%0A%3E%0A%3E%20%60%60%60bash%0A%3E%20masters%20%3D%20gentoo%0A%3E%20auto-sync%20%3D%20false%0A%3E%20%60%60%60%0A%0A%3E%20%60%2Fetc%2Fportage%2Frepos.conf%2Flocalrepo.conf%60%0A%3E%0A%3E%20%60%60%60bash%0A%3E%20%5Blocalrepo%5D%0A%3E%20location%20%3D%20%2Fvar%2Fdb%2Frepos%2Flocalrepo%0A%3E%20%60%60%60%0A

If you do not like what this rule does, you can disable it.￼

**Comment by cjwijtmans** [comment]
yes, I added the ignore warning rule in my document.

> Rationale: Some markdown parsers will treat two blockquotes separated by one or more blank lines as the same blockquote, while others will treat them as separate blockquotes.

However I do not think its a valid reason as there are plenty of markdown viewers that have no problem with recognising the separate block quotes. This should be the fault of the markdown viewer, not the document.

**Comment by DavidAnson** [comment]
If a document looks bad (or inconsistent) because of a bug or quirk of the renderer, the document still looks bad. This rule exists to help prevent that from happening. If the renderers you use all behave consistently or this is not something you care about, you can disable the rule. But I claim the rule has a valid purpose and therefore am not inclined to remove it.

**Comment by cjwijtmans** [comment]
> If a document looks bad (or inconsistent) because of a bug or quirk of the renderer, the document still looks bad. This rule exists to help prevent that from happening. If the renderers you use all behave consistently or this is not something you care about, you can disable the rule. But I claim the rule has a valid purpose and therefore am not inclined to remove it.

There is nothing wrong with the document. If the renderer is not smart enough to properly render it is the fault of the renderer, even i had 0 issues with my renderer. It should not be the job of the linter to protect some obscure buggy renderer and a bug should be reported at the renderer instead.

**Comment by DavidAnson** [comment]
In theory, yes. In practice, I definitely want to know if my document will render inconsistently.

Also, it is not always a question of bugs in the renderer; before CommonMark was defined, the standards and implementations around Markdown varied quite a bit. Two different implementations could deliberately handle this differently (i.e., not a bug). These days, I prefer tools that obey the CommonMark specification, but there are plenty of older tools that do not.

**Comment by cjwijtmans** [comment]
Should it not be an opt-in rather than an opt-out then? Also i do not agree on it not being a bug, if the implementation is bad it is a bad implementation.

**Comment by DavidAnson** [comment]
By convention, all rules are enabled by default.

**Comment by DavidAnson** [comment]
MD028 looks to be disabled about 5% of the time. That is not very often, so I plan to leave it on by default as it is today.

https://dlaa.me/blog/post/markdownlintanalyzeconfig
